SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -193,17 +171,93 @@ bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchC
 			return false;
 		}
 
+		if (bGenerateCloudConfig)
+		{
+			return ConvertToClassicConfig(LaunchConfigPath, InLaunchConfigDescription);
+		}
+
 		return true;
 	}
 
 	return false;
 }
 
-bool ValidateGeneratedLaunchConfig(const FSpatialLaunchConfigDescription& LaunchConfigDesc, const FWorkerTypeLaunchSection& InWorker)
+bool ConvertToClassicConfig(const FString& LaunchConfigPath, const FSpatialLaunchConfigDescription* InLaunchConfigDescription)
 {
-	const USpatialGDKSettings* SpatialGDKRuntimeSettings = GetDefault<USpatialGDKSettings>();
+	// The new runtime binary handles the config conversion. We just need to pass it the parameters.
+	// All `runtime_flags` are converted to `legacy_flags` in the conversion process.
+	// `max_concurrent_workers` is also converted to a legacy flag.
+	// The output config file will be moved to the same location as the generated standalone config file.
+	// The output config file will have `_classic` appended to it to distinguish it's usage in classic platform (spatial/spatiald/cloud).
```

Comment:
Need to change this to `Cloud` as we're now making a `Cloud` config and a `Local` config, rather than having two of both and also `_classic` versions